GNUmakefile

```diff
@@ -83,7 +83,7 @@ docker-plugins:
 
 .PHONY: operator-ui
 operator-ui: ## Fetch the frontend
-	go generate ./core/web
+	cd operator_ui && go build -o install install.go && ./install
```

Comment:
```suggestion
	go run ./operator_ui/install.go
```

Reply:
This doesn't work because the path resolution is relative to the binary location, `go run` puts that in `tmp`

Reply:
What path resolution? Can't we control that another way/

Reply:
It works the same way the old shell script did, where it resolves paths relative to the binary location
```sh
gitRoot="$(dirname -- "$0")/../"
```

Reply:
Why is that better to do than to use the working directory?

Reply:
I would say that its less likely that a user accidentally executes in a way where the paths are not correctly resolved. If you feel strongly about this then I can change it to use cwd instead. I used to use cwd, but users would execute it from some other directory within the repo causing resolution errors. I find git resolution to be the most reliable one, but that fails in docker contexts since `.git` isnt copied over during image builds.

Reply:
I'm not sure I grok the full picture, but can we use `go:embed` to reference files and dirs to avoid this issue?

Reply:
I'm not sure what you mean. Would you be able to push what you're proposing? Otherwise, I think it's fine as-is. We use a makefile invocation for `modgraph` too.

Reply:
`go:embed` is read-only, so it probably doesn't fit for everything we are doing it, but it would make sense for the tag file. I'm just trying to find an alternative that doesn't depend on the location of the installed binary. That seems like a hacky limitation to accept.

Reply:
Wait, can we take two steps back here? What is actually going on with `go run`? because we have many instances of things like this which apparently work fine:
```go
//go:generate go run ./generation/generate/wrap.go ../../contracts/solc/...
```

Reply:
In this case, I think the single `go run` invocation could live inside of the `main.go` so that it is effectively encoding the path reference in the source like embed would.